docs/CONTRIBUTORS

```diff
@@ -176,3 +176,4 @@ Yves Mathieu
 Zhanglei Wang
 Zixi Li
 أحمد المحمودي
+Julian Daube
```

Comment:
Please put into sorted position and fix the missing newline, then I think tests will pass, thanks.

Reply:
done